.github/workflows/build-dotnet.yml

```diff
@@ -8,8 +8,10 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
-  build:
+  cloud-builds:
```

Comment:
Just curious, why did you use the moniker `cloud-builds` here? I'm guessing it has to do with the hosted environment and you wanted to index / group on that.

Reply:
Uhh...you know, it seems like pretty poor naming. I guess I did it because right now it only comprises of cloud-based products, but I'm not sure how that'll fit when we work in GHES.

Reply:
Maybe `sdk-builds`? `builds` would probably be fine too; I think I changed it because I originally got confused and thought it might have been a reserved Actions keyword.